src/applications/vaos/new-appointment/components/ReviewPage/ReasonForAppointmentSection.jsx

```diff
@@ -25,15 +34,19 @@ export default function ReasonForAppointmentSection({ data }) {
             </span>
           </div>
           <div>
-            <Link
-              to={newAppointmentFlow.reasonForAppointment.url}
+            <VaLink
```

Comment:
Let's use the `va-link` for consistency
```suggestion
            <va-link
```

Reply:
@jenniemc Yeah, I had used the `VaLink` version because I had thought we'd need the react bindings, but for the most part we do not. Will change.